src/wallet_new.ts

```diff
@@ -3,6 +3,7 @@ import * as B64js from 'base64-js';
 import * as crypto from 'crypto';
 import jwkToPem, { JWK } from 'jwk-to-pem';
 import Arweave from 'arweave';
+import mnemonicKeys from 'arweave-mnemonic-keys';
```

Comment:
This results in mnemonicKeys being `undefined` for me, this fixes it:
```suggestion
import * as mnemonicKeys from 'arweave-mnemonic-keys';
```

Reply:
Fixed here: 9bbcb0033d77d0a1263fbe11cefa41fcc6013222